### PR TITLE
Remove parser

### PR DESCRIPTION
**Pull request by Mec-iS**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -24,3 +24,4 @@ thespian==3.9.2
 Werkzeug==0.14.1
 PyYAML==4.2b1
 -e git+https://github.com/HTTP-APIs/hydra-python-core#egg=hydra_python_core
+-e git+https://github.com/HTTP-APIs/hydra-python-core#egg=hydra_python_core
```
Comment (xadahiya) [inline requirements.txt]: This should be the parser repo

**Comment by xadahiya** [comment]
One more thing, .travis.yml needs to be updated to include hydraspecs module in install or tests will fail

**Comment by Mec-iS** [comment]
> One more thing, .travis.yml needs to be updated to include hydraspecs module in install or tests will fail

CI should install from `requirements.txt`

**Comment by xadahiya** [comment]
For some reason `pip install -e` fails with Travis. Found this solution in stack overflow

**Comment by Mec-iS** [comment]
`pip install -r requirements.txt` from the project directory?

**Comment by xadahiya** [comment]
I guess anything with `-e` in requirements.txt will fail but only for travis. Locally it works fine

**Comment by Mec-iS** [comment]
Probably we need to specify a version for the parser and core repos

**Comment by xadahiya** [comment]
Yeah that might work

**Comment by Mec-iS** [comment]
https://www.simonmweber.com/2014/11/23/git-pip-requirements.html

**Comment by Mec-iS** [comment]
`pip` command fixed.

> Also the parser reference in cli.py needs to be updated.

Please open a issue for this

**Pull request merged**